### ISSUE TITLE
Update Swiper Version

### ISSUE THREAD

**Issue by korgon** [searchspring/snap#495]
Look into Updating to latest Swiper version. Additional testing needs to be done - testing was done in Jest but may need to be moved to Cypress in order to handle ESM changes.

**Comment by korgon** [comment]
This was attempted, but due to no CJS support it breaks our Jest tests. Will have to try when either Jest supports ESM completely or when Swiper packages CJS again.

**Comment by chrisFrazier77** [comment]
update is complete on update-swiper branch. still waiting on cjs support from swiper

**Comment by korgon** [comment]
We may have issues updating sites with some sites using Swiper in unexpected ways:
https://github.com/search?q=org%3Asearchspring-implementations+SwiperCore&type=code

**Comment by korgon** [comment]
Merged into the `templates` branch.

**Comment by korgon** [comment]
v11 appears to have an issue with TS4 - planning on launching bundle recs with v9, then merging into `templates` branch, after which we can attempt to update again.
